tests/test_app.py

```diff
@@ -533,6 +533,9 @@ def get_lines_from_retrieval_list(rv):
 
 def get_num_ballots_from_retrieval_list(rv):
     lines = get_lines_from_retrieval_list(rv)
+
+    for line in lines:
+        print(line)
```

Comment:
@umbernhard did you mean to leave these here?

Reply:
Ope, no I did not.

Reply:
Fixed in #576